src/main/java/com/ait/lienzo/client/core/image/ImageStrips.java

```diff
@@ -0,0 +1,195 @@
+package com.ait.lienzo.client.core.image;
+
+import com.ait.lienzo.client.core.Context2D;
+import com.ait.tooling.common.api.java.util.function.Supplier;
+import com.ait.tooling.nativetools.client.collection.NFastStringMap;
+
+public class ImageStrips {
```

Comment:
Class formatted as Kie-Group class, not Lienzo's one. If we want to merge changes to upstream it will be an issue in the future.